cli/script/command-executor.ts

```diff
@@ -101,9 +101,36 @@ export var confirm = (): Promise<boolean> => {
 }
 
 function accessKeyAdd(command: cli.IAccessKeyAddCommand): Promise<void> {
-    return sdk.addAccessKey(command.description)
+    return sdk.addAccessKey(command.friendlyName, command.maxAge)
         .then((accessKey: AccessKey) => {
-            log("Successfully created a new access key" + (command.description ? (" \"" + command.description + "\"") : "") + ": " + accessKey.name);
+            log(`Successfully created a new access key "${command.friendlyName}": ${accessKey.name}\n(Expires: ${new Date(accessKey.expires).toString()})`);
+        });
```

Comment:
Should we add some verbiage here to ensure that devs know they need to save off the key somewhere since it won't be displayed anywhere in the CLI?